sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -41,6 +44,7 @@ def __init__(self, endpoint: str, **kwargs) -> None: # pylint: disable=missing-c
             **kwargs
         )
 
+    @distributed_trace
```

Comment:
Do we need distributed_trace for both methods?

It looks to me one just calls the other.

Reply:
Also, should we trace the methods to get token?

Let's check the behavior in other libraries e.g. key vault.

Reply:
I don't find a similar usage in KV. For the case here, since this client is not public, I think it's not necessary to add decorators on both.